app/scripts/inpage.js

```diff
@@ -34,7 +34,10 @@ cleanContextForImports();
 import log from 'loglevel';
 import { WindowPostMessageStream } from '@metamask/post-message-stream';
 import { initializeProvider } from '@metamask/providers/dist/initializeInpageProvider';
+import { v4 as uuid } from 'uuid';
```

Comment:
This will increase the size of the injected script by <=11kb unless we tree shake.